erts/emulator/test/op_SUITE.erl

```diff
@@ -450,6 +457,225 @@ usec_to_seconds(Usec) when is_integer(Usec) ->
     Sec = Usec div 1000000,
     Sec rem 1000000.
 
+range_tests(_Config) ->
+    {MinSmall,MaxSmall} = small_limits(),
+
+    small = range(-1 bsl 64),
```

Comment:
I was a bit confused by `small_limits/0` and then seeing that `-1 bsl 64` is somehow `small`. Maybe use `lesser` / `greater` instead?

Reply:
Good idea. Changed.